lib/nerves/package/artifact.ex

```diff
@@ -56,7 +56,6 @@ defmodule Nerves.Package.Artifact do
     else
       base_dir(pkg)
       |> Path.join(name(pkg, toolchain))
-      |> protocol_vsn(pkg)
```

Comment:
I'm not sure I understand why this whole function is no longer needed. It seems like it would still be useful for manipulating the `build_path` for toolchains and systems.

Reply:
configuration for the build_paths for the compiler should be done in the compiler instead of package then. This only removes some code which would look in a different area for code to support the compiler packages in the long deprecated `nerves_toolchain` and `nerves_system` apps.